ext/gd/gd.stub.php

```diff
@@ -626,6 +669,11 @@ function imagejpeg(GdImage $image, $file = null, int $quality = -1): bool {}
 /** @param resource|string|null $file */
 function imagewbmp(GdImage $image, $file = null, ?int $foreground_color = null): bool {}
 
+#ifdef HAVE_GD_HEIF
+/** @param resource|string|null $file */
+function imageheif(GdImage $image, $file = null, int $quality = -1, ?int $codec = null, ?string $chroma = null): bool {}
```

Comment:
I'm aware that this matches the signature of libgd's heif writers, but I'm not sure if we should add full support for codes (and chroma). So far, all our image writers support only one codec per container format, and these are treated as synonymous (although Windows BMP can store JPEG encoded images, for instance). So supporting both HEVC and AV1 for HEIF containers might be confusing for users, albeit it is still limiting (perhaps it's time for libgd to consider introducing a clear separation between containers and codecs). Furthermore, in practice the only relevant combination appears to be HEIF with HEVC (aka. HEIC).